fearless/ApplicationLayer/SoraCard/SCService/SCKYCService.swift

```diff
@@ -29,14 +30,12 @@ final class SCKYCService {
 
     internal let client: SCAPIClient
     private let payWingsOAuthClient: PayWingsOAuthSDK.OAuthServiceProtocol
-    let apiKey: String
 
     init(client: SCAPIClient) {
         self.client = client
 
-        let domain = "soracard.com"
-        let apiKey = SoraCardKeys.apiKey
-        self.apiKey = apiKey
+        let domain = SoraCardCIKeys.domain
+        let apiKey = SoraCardCIKeys.apiKey
         PayWingsOAuthClient.initialize(environmentType: .TEST, apiKey: apiKey, domain: domain)
```

Comment:
can you initialize client dynamically based on which type of app version used ? production/debug